app/scripts/modules/core/pipeline/config/stages/tagImage/tagImageExecutionDetails.html

```diff
@@ -0,0 +1,35 @@
+<div ng-controller="TagImageExecutionDetailsCtrl as tagImageCtrl">
+  <execution-details-section-nav sections="configSections"></execution-details-section-nav>
+  <div class="step-section-details" ng-if="detailsSection === 'tagImage'">
+    <div class="row">
+      <div class="col-md-12">
+        <dl class="dl-narrow dl-horizontal">
+          <dt ng-if="stage.context.target.imageName">Image Name</dt>
+          <dd ng-if="stage.context.target.imageName">{{stage.context.target.imageName}}</dd>
+          <dt ng-if="stage.context.target.regions">Regions</dt>
+          <dd ng-if="stage.context.target.imageName">{{stage.context.target.regions.join(', ')}}</dd>
+        </dl>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-12">
+        <strong>Updated Tags</strong>
+        <dl class="dl-narrow dl-horizontal">
+          <div ng-repeat="(key, val) in stage.context.tags" ng-if="val !== null">
+            <dt>{{key | robotToHuman}}</dt>
```

Comment:
Do we want to format the key on the tag? Might be confusing for users to see "some tag" in the UI here and "someTag" elsewhere...